app/controllers/api/v1/authentication_controller.rb

```diff
@@ -29,6 +27,20 @@ def signup
     end
   end
 
+  # POST api/v1/forgot_password
+  def forgot_password
+    @user = User.find_by!(email: params[:email])
+    # sends reset password instructions to the user's mail if exists
+    @user.send_reset_password_instructions
+    render json: { message: "password reset instructions sent to #{@user.email}" }
+  end
+
+  # GET /public_key.pem
+  def public_key
+    public_key = File.open(Rails.root.join("config", "public.pem"), "r:UTF-8")
+    send_file public_key
```

Comment:
What about making this a json so that this will be easier to consume at client level

Reply:
Yes!! Would be easier..